pkg/apis/devops/v1alpha1/s2ibinary_types.go

```diff
@@ -20,6 +20,23 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	ResourceKindS2iBinary         = "S2iBinary"
+	ResourceSingularServicePolicy = "s2ibinary"
+	ResourcePluralServicePolicy   = "s2ibinaries"
+)
+
+const (
+	StatusUploading    = "Uploading"
+	StatusReady        = "Ready"
+	StatusUploadFailed = "UploadFailed"
+)
+
+const (
+	S2iBinaryFinalizerName = "s2ibinary.finalizers.kubesphere.io"
+	S2iBinaryLabelKey      = "s2ibinary-name.kubesphere.io"
```

Comment:
need to add specified Label and Annotation to https://tower.im/teams/637817/repository_documents/8362/.

Reply:
add